### ISSUE TITLE
Evaluation Pipeline of TinyFace

### ISSUE THREAD

**Issue by zcy8123878** [mk-minchul/AdaFace#18]
Hello, 

I wonder whether you have finetuned the model on the training set of TinyFace before testing as in https://arxiv.org/pdf/1811.08965.pdf, or directly applied it on the testing set without finetuning.

**Comment by mk-minchul** [comment]
Hi zcy8123878
We did not fine-tune on the Tinyface training data.

**Comment by zcy8123878** [comment]
Thanks for your reply. 

I encountered a little difficulty when evaluating the pre-trained model on TinyFace.

Specifically,  
1) I downloaded 'adaface_ir101_ms1mv2.ckpt' and extracted features following the pipeline of TinyFace; 
Here is my feature extracting code:
[extract_tinyface_features.zip](https://github.com/mk-minchul/AdaFace/files/8851458/extract_tinyface_features.zip)

2) Then, I evaluated the extracted features with TinyFace official code https://qmul-tinyface.github.io/.

However, the Rank-1 value is a little lower than the released result. Therefore, I think there is something wrong with my evaluating pipeline. Suggestions are welcome if I make any mistakes.
